application.md

```diff
@@ -1,5 +1,141 @@
 # Questionnaire
 
+## Why does your org want to participate in Google Summer of Code?
+
+OpenAstronomy is a collection of projects united by a common goal to foster open
```

Comment:
I don't think we ever actually answer the question as posed.  The answer here answers why the disparate groups are applying *together*, but doesn't actually say why we want to participate.  Perhaps lead off with something like this:

OpenAstronomy would like to be a part of Google Summer of Code to reach a new pool of contributors, develop our communities, and help the member projects collaborate better.  Some of our member projects' past experience with GSoC shows that it provides a boost to areas which are in need of intensive attention, as well as adding members to the community who can turn out to be valuable long-lasting contributors.  We would like to help bring these experience to a wider variety of projects, all united by the goal of fostering open source development of astronomy software.

We are submitting this application together...